.github/workflows/ci.yml

```diff
@@ -13,28 +13,77 @@ on:
 name: CI
 
 jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+        toolchain: [ "1.65.0", "stable" ]
+    runs-on: ${{ matrix.os }}
+    needs: lint
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ matrix.toolchain }}
+      - name: Install cargo-binstall
+        uses: taiki-e/install-action@cargo-binstall
+      - name: Install cargo-make
+        run: |
+          cargo binstall -y cargo-make
+          cargo make --version
+          cargo make --list-all-steps
```

Comment:
```suggestion
      - name:: install cargo make
        uses: taiki-e/install-action@370d6b7a85900451456cc58b71ebdbe965d20d8e
        with:
          tool: cargo-make
```

Reply:
Could you suggest these changes again, because can't apply them to the outdated file

Reply:
Yeah - I fixed these with a proper push